duckduckgo/Android
# Add light theme

Task/Issue URL: https://app.asana.com/0/361428290920652/814618797160185
Tech Design URL: https://app.asana.com/0/361428290920652/814618797160182

**Description**:
Add a light theme to the app as per designs at https://app.zeplin.io/project/59b98582fe3bbb034e7ec10f/dashboard?seid=5b6dfa95543fee608da303d4.

The widget and experiments will be submitted as separate PRs.

**Steps to test this PR**:

**TEST ONE: Theme selection and visuals**
1. Run the app and note that the dark theme still looks ok
1. Go to Settings, toggle the "Light theme" on and note that the theme is updated
1. Check the various screens and make sure that the light theme renders fine

**TEST TWO: Ensure theme selection persists**
Continuing from Test One...
1. Kill the app and relaunch
1. Ensure the light theme is still showing
1. Go to settings, switch to the dark theme
1. Kill the app and relaunch
1. Ensure the dark theme is still showing

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
